bom/narayana-test-bom/pom.xml

```diff
@@ -10,9 +10,10 @@
     <groupId>org.jboss.narayana</groupId>
     <artifactId>narayana-all</artifactId>
     <version>7.2.3.Final-SNAPSHOT</version>
+    <relativePath>../../pom.xml</relativePath>
   </parent>
   <groupId>org.jboss.narayana</groupId>
-  <artifactId>narayana-bom-test</artifactId>
+  <artifactId>narayana-test-bom</artifactId>
```

Comment:
Thanks @jmfinelli for your PR, I can understand the reasoning for this change. But this means that a new artifact will be released on maven central (new name). Are you fine with that? I think this can impact users and should go with a minor or major release.